src/Fantomas.Tests/ComputationExpressionTests.fs

```diff
@@ -2329,3 +2329,77 @@ let generateUnionCases =
             |> AsyncResult.foldResult id (fun _ -> []))
         "Incomplete pattern matches on this expression. For example"
 """
+
+[<Test>]
+let ``keep newline before multiline expression, 1463`` () =
```

Comment:
Maybe go for `keep newline before multiline SynExpr.JoinIn, 1463`.

Reply:
Done :)